pymc/distributions/continuous.py

```diff
@@ -1190,6 +1190,13 @@ def dist(cls, alpha=None, beta=None, mu=None, sigma=None, sd=None, *args, **kwar
 
         return super().dist([alpha, beta], **kwargs)
 
+    def get_moment(rv, size, alpha, beta):
+        alpha, beta = at.broadcast_arrays(alpha, beta)
```

Comment:
```suggestion
```
You don't need this line. The operation below will automatically broadcast the arrays.